apps/site/src/components/NavBar/NavBar.module.scss

```diff
@@ -0,0 +1,40 @@
+@use "@/lib/styles/variables.scss" as variables;
+
+$red: #ff0000;
```

Comment:
Question: is there a reason why the `$red` variable is in this file and not in `variables.scss`?

Reply:
> Question: is there a reason why the `$red` variable is in this file and not in `variables.scss`?

The `$red` in `variables.scss` has a different hex value which was why I put it in this file, but I will make another variable called like `$navbar-red` in `variables.scss`